### PR TITLE
fix(v2): remove obsolete iframe attributes

### PR DESCRIPTION
**Pull request by lex111**
## Motivation

The `frameborder` and `scrolling` attributes on the `iframe` element is obsolete. Use CSS instead.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

No UI changes.

### CODE REVIEW

File: website/src/pages/styles.module.css [hunk 1]
```diff
@@ -125,6 +125,11 @@
   vertical-align: sub;
 }
 
+.indexCtasGitHubButton iframe {
```
Comment (yangshun) [inline website/src/pages/styles.module.css]: Can we add an id or a class to the `iframe`?

Comment (lex111) [reply website/src/pages/styles.module.css]: Done :heavy_check_mark: 

BTW, what does "Ctas" mean? Is this by chance not "Stars"?

Comment (yangshun) [reply website/src/pages/styles.module.css]: CTA means Call-to-action lol. Things get lost in translation when we use acronyms with bad casing.

Comment (lex111) [reply website/src/pages/styles.module.css]: Oops :face_with_head_bandage:, I would rename it to something else, but this is not an important thing.

**Pull request merged**